public/app/features/explore/state/query.ts

```diff
@@ -447,7 +447,8 @@ export const runQueries = (
       cache,
       supplementaryQueries,
     } = exploreItemState;
-    let newQuerySub;
+    let newQuerySource: Observable<ExplorePanelData>;
+    let newQuerySubscription: SubscriptionLike | undefined;
```

Comment:
we never try to use it before assignment, so 
```suggestion
    let newQuerySubscription: SubscriptionLike;
```
works as well

Reply:
Updated in f4190d1243f4431dabfa091ed71551ac95eb1b55